### ISSUE TITLE
Implement HTMLTableElement's head sections operations

### ISSUE THREAD

**Issue by nox** [servo/servo#9023]
https://html.spec.whatwg.org/multipage/tables.html#htmltableelement

```
  attribute HTMLTableSectionElement? tHead;
  HTMLTableSectionElement createTHead();
  void deleteTHead();
```

Files:
 * [components/script/dom/htmltableelement.rs](https://github.com/servo/servo/blob/master/components/script/dom/htmltableelement.rs)
 * [components/script/dom/webidls/HTMLTableElement.webidl](https://github.com/servo/servo/blob/master/components/script/dom/webidls/HTMLTableElement.webidl)

Cc @GuillaumeGomez

**Comment by KiChjang** [comment]
His name is @GuillaumeGomez here on GitHub, by the way :)

**Comment by nox** [comment]
Oh, right… I'm stupid.

**Comment by GuillaumeGomez** [comment]
Thanks @KiChjang. :D
Well, thanks too @nox haha.

**Comment by dzbarsky** [comment]
https://github.com/servo/servo/pull/8745, just needs some tests :)

**Comment by nox** [comment]
@dzbarsky Oh… Forgot that you had written some things related to it. @GuillaumeGomez Want to write the tests, or do you want me to find something else for you?

**Comment by GuillaumeGomez** [comment]
@nox: I'd prefer something else if you don't mind. ^^
Le 19 déc. 2015 3:21 PM, "Anthony Ramine" <notifications@github.com> a
écrit :

**Comment by nox** [comment]
@GuillaumeGomez Sure.

**Comment by KiChjang** [comment]
Fixed by #9980.
